tests/logging-utils/test_compat_with_typing_Generic.py

```diff
@@ -1,16 +1,18 @@
 import logging
-from typing import GenericMeta, TypeVar
+import sys
 
-from eth_utils import HasLoggerMeta
+import pytest
 
-T = TypeVar("T")
+from eth_utils import HasLoggerMeta
 
 
-class HasLoggerCompatWithGeneric(metaclass=HasLoggerMeta.meta_compat(GenericMeta)):
-    ...
+@pytest.mark.skipif(sys.version_info[:2] == (3, 7), reason="GenericMeta not present in Python 3.7+")
```

Comment:
If it's missing in 3.7+, then the skipif could be something like:
```suggestion
@pytest.mark.skipif(sys.version_info[:2] >= (3, 7), reason="GenericMeta not present in Python 3.7+")
```